.github/workflows/image-build.yaml

```diff
@@ -356,20 +378,20 @@ jobs:
       - uses: actions/checkout@v4
 
       - uses: actions/setup-go@v5
-        if: ${{ matrix.image.bpf_build_wrapper == 'go' }}
+        if: ${{ matrix.image.build_language == 'go' }}
         with:
           go-version: "1.21"
 
       - uses: sigstore/cosign-installer@v3.3.0
 
       - uses: actions/checkout@v4
-        if: ${{ matrix.image.bpf_build_wrapper == 'rust' }}
+        if: ${{ matrix.image.build_language == 'rust' }}
```

Comment:
Wait we only need libbpf if there's bpf code being built! 

I think we want to leave this as bpf_build_wrapper

Reply:
And I'm pretty sure we only need it for rust/aya based bpf builds 🤔

Reply:
Good ctach. I thought I saw it elsewhere, but just re-looked and only needed for BPF builds.